client/client.go

```diff
@@ -88,10 +91,25 @@ func (c *Client) Mkdir(ctx context.Context, name string, all bool) error {
 	return parseErr(reply.Error)
 }
 
+func (c *Client) Stats(ctx context.Context) (string, error) {
+	reply, err := c.statsClient.ContainerInfo(ctx, &pb.ContainerInfoRequest{})
+
+	if err != nil {
+		return "", err
+	}
+	if reply.Error != "" {
+		return reply.Info, errors.New(reply.Error)
```

Comment:
Is there a chance we can have only one basic Error type in pb for both file.pb and stats.pb and further pb?